tools/tf2openapi/types/tfsavedmodel.go

```diff
@@ -18,25 +18,23 @@ type TFSavedModel struct {
 	MetaGraphs [] TFMetaGraph
 }
 
-func NewTFSavedModel(model *pb.SavedModel) (TFSavedModel, error) {
+func NewTFSavedModel(model *pb.SavedModel, sigDefKey string) (TFSavedModel, error) {
 	tfSavedModel := TFSavedModel{
 		MetaGraphs: []TFMetaGraph{},
```

Comment:
It's unclear to me whether `MetaGraphs` have to be a list now, since the size is guaranteed to be at most 1, correct?

Reply:
I left `MetaGraphs` (and `SignatureDefs` in `tfmetagraph.go`) as lists in case in the future, it is desired to convert multiple MetaGraphs or SignatureDefs to an internal model. But for now, since we let the user choose exactly one SignatureDef to convert, I agree that neither should be lists.

Reply:
Update: We are now representing the full model in this internal model. Thus, the user's choice for the SignatureDef to convert affects the OpenAPI output but not the internal model construction.